config/locales/ar/shared.yml

```diff
@@ -6,7 +6,7 @@ ar:
       prefix: 'تصفَّح:'
       title: تصفَّح GOV.UK
     get_emails: احصل على رسائل بريد إلكتروني عن هذا الموضوع
-    language_direction: ltr
+    language_direction: rtl
```

Comment:
Nitpick: could your commit message link to the resource you used to determine which languages are RTL?

Reply:
Sure thing, have added some more detail to the commit message.